src/sys/ptrace.rs

```diff
@@ -116,6 +116,10 @@ libc_bitflags! {
         /// Stop tracee when a SECCOMP_RET_TRACE rule is triggered. See `man seccomp` for more
         /// details.
         PTRACE_O_TRACESECCOMP;
+        /// Send a SIGKILL to the tracee if the tracer exits.  This is useful
+        /// for ptrace jailers to prevent tracees from escaping their control.
+        #[cfg(target_os = "linux")]
```

Comment:
This should be available on Android as well. Please keep the OSes listed in alphabetical order here.

Reply:
good catch!  updated diff.